app/components/autocomplete/emoji_suggestion/emoji_suggestion.js

```diff
@@ -82,7 +82,7 @@ export default class EmojiSuggestion extends Component {
                 return;
             }
 
-            if (this.props.emojis !== nextProps.emojis) {
+            if (this.props.emojis === nextProps.emojis && this.matchTerm.length) {
```

Comment:
Don't we also want to run `handleFuzzySearch` when `this.props.emojis !== nextProps.emojis`?